src/lib/features/taskcluster_proxy.js

```diff
@@ -37,9 +37,9 @@ class TaskclusterProxy {
     if (task.claim.credentials.certificate) {
       cmd.push('--certificate=' + task.claim.credentials.certificate);
     }
-    const cert = JSON.parse(task.claim.credentials.certificate);
-    cmd.push(`--task-id=${task.status.taskId}`);
```

Comment:
Is the `--task-id` parameter not needed anymore or this was a typo?

Reply:
Correct, it's not needed since we are providing the information the proxy might need from the task (its scopes).